RNAwesomeCardIO.podspec

```diff
@@ -10,10 +10,10 @@ Pod::Spec.new do |s|
   s.homepage           = package["homepage"]
   s.authors            = { package["author"]["name"] => package["author"]["email"] }
   s.source             = { :git => s.homepage, :tag => "v#{s.version}" }
-  s.source_files       = ["ios/*.{h,m}", "ios/CardIO/*.h"]
+  s.source_files       = "ios/*.{h,m}"
   s.platform           = :ios, "8.0"
   s.frameworks         = "Accelerate", "AVFoundation", "AudioToolbox", "CoreMedia", "CoreVideo", "MobileCoreServices", "OpenGLES", "QuartzCore", "Security", "UIKit"
   s.libraries          = "c++"
```

Comment:
now that OpenCV is gone, do we need the `s.libraries` declaration?

It's listen in the CardIO Podspec: https://github.com/card-io/card.io-iOS-SDK/blob/master/CardIO.podspec#L15

Reply:
I guess this whole podspec can be deleted. It's useless now 🤔

Reply:
Oh wait I was misunderstanding. My bad.